app/src/main/res/drawable/metadata_label_rounded_background.xml

```diff
@@ -3,5 +3,5 @@
   <solid
     android:color="@color/white"/>
   <corners
-    android:radius="2dp"/>
+    android:radius="0dp"/>
```

Comment:
oooo you can actually delete this file now since we don't use this rounded background anywhere anymore!